exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestSchema.java

```diff
@@ -30,6 +30,7 @@
 
   private final View view;
 
+  // TODO: inject ServiceInstanceSpec
```

Comment:
It is not injectable in the container sense of things (service creates it manually), you will have to pass it.